infra/helper.py

```diff
@@ -524,7 +524,7 @@ def _get_fuzz_targets(project_name):
   fuzz_targets = []
   for name in os.listdir(_get_output_dir(project_name)):
     path = os.path.join(_get_output_dir(project_name), name)
-    if os.path.isfile(path) and os.access(path, os.X_OK):
+    if os.path.isfile(path) and os.access(path, os.X_OK) and not name.startswith('afl-'):
```

Comment:
can you move "not name.startswith('afl-'):" to its own if before line 526.

if not name.startswith('afl-'):
  continue